src/clients/db/Repository.ts

```diff
@@ -1,4 +1,4 @@
-/* eslint "@typescript-eslint/camelcase": "off" */
+/* eslint "@typescript-eslint/camelcase": "off", "@typescript-eslint/no-unused-vars": "off" */
```

Comment:
In general we want to not have unused variables in our code, however, in this file we are restructuring generic objects and ensuring we don't copy over some ids, which is why I have ignored it in this file and this file only.

Reply:
Add this rule https://eslint.org/docs/rules/no-unused-vars#ignorerestsiblings to eslintrc instead.
` { "ignoreRestSiblings": true }`
On object destructuring is always safe to assume that we just want to get rid of some props.